lib/routes/brain.rb

```diff
@@ -27,30 +27,29 @@ def self.registered(slapi)
           # Saves into brain as Plugin: plugin name (hash), key, value
           slapi.save(params[:plugin], params[:key], params[:value])
           status 200
-          { 'message' => 'it worked' }.to_json
         end
 
         # Handles a POST request for '/v1/delete'
         #
         # @param [Hash] params the parameters sent on the request
         # @option params [String] :plugin Utilizes the plugin name as Hash Name
         # @option params [String] :key Key you wish to delete (will delete value as well)
-        # @return [String] status or message "it worked"
+        # @return [Integer] status
         slapi.post '/v1/delete' do
           raise 'missing plugin name' unless params[:plugin]
           raise 'missing key' unless params[:key]
 
           # Saves into brain as Plugin: plugin name (hash), key
           slapi.delete(params[:plugin], params[:key])
           status 200
```

Comment:
Yea, I need to create issues for the codes specifically. I was just trying to clean up all the "it works" items. Really need to make the returns more useful.